scripts/format-stylish.sh

```diff
@@ -18,6 +18,6 @@ fi
 # Check Haskell files with stylish-haskell
 echo "Formatting Haskell source files with stylish-haskell version ${stylish_haskell_required_version}"
 # shellcheck disable=SC2016
-if ! git ls-files --exclude-standard --no-deleted --deduplicate '*.hs' | xargs -L 1 sh -c 'echo "$0" && stylish-haskell -i -c .stylish-haskell.yaml "$0"'; then
+if ! git ls-files --exclude-standard --no-deleted --deduplicate '*.hs' | xargs -L 50 stylish-haskell -i -c .stylish-haskell.yaml; then
```

Comment:
Does batching make a difference?

Reply:
Yes. Massive. In fact, it's the only thing that made a difference.